app/views/admin/loans/questionnaires/_questionnaire_group.html.slim

```diff
@@ -84,7 +84,8 @@
               = f.input_field :"#{question.attribute_sym}[number]", as: :decimal,
                 value: response.number
 
-            - tags = %w(table tbody tr td b i u ol ul p)
+            - tags = %w(table tbody tr td b i u ol ul li p)
+            = response.text
```

Comment:
i believe this is a debugging line, you'll want to remove it

Reply:
thanks for the find